import-chrome/src/import-chrome.cpp

```diff
@@ -133,11 +133,30 @@ int main( int argc, char** argv )
     printf( "\33[2KParsing...\r" );
     fflush( stdout );
 
+    struct PidTid {
+      uint64_t tid;
+      uint64_t pid;
+      uint64_t both; // fake thread id, unique within Tracy
+    };
+
+    std::vector<PidTid> tids;
     std::vector<tracy::Worker::ImportEventTimeline> timeline;
     std::vector<tracy::Worker::ImportEventMessages> messages;
     std::vector<tracy::Worker::ImportEventPlots> plots;
     std::unordered_map<uint64_t, std::string> threadNames;
 
+    const auto getTid = [&](uint64_t pid, uint64_t tid) -> uint64_t {
```

Comment:
This has to be better named. There are two different meanings for `tid`. At the same time, `tid` and `both` are used as aliases.